Makefile

```diff
@@ -90,10 +90,10 @@ else
 	$(call php, drush si $(PROFILE_NAME) --db-url=$(DB_URL) --account-name=$(ADMIN_NAME) --account-mail=$(ADMIN_MAIL) --account-pass=$(ADMIN_PW) -y --site-name="$(SITE_NAME)" --site-mail="$(SITE_MAIL)" install_configure_form.site_default_country=FR install_configure_form.date_default_timezone=Europe/Paris)
 endif
 ifneq ($(strip $(MODULES)),)
```

Comment:
In other PR https://github.com/skilld-labs/skilld-docker-container/pull/232#discussion_r406734049 @andypost mentioned we are getting rid of default_content so IMO we should remove this part

Mb we need a new issue to explore new ways to deal with content import.

Reply:
Let's keep changes in own scope, we anyway need working d9-compatible release with DC approach